app-arch/par2cmdline-turbo/par2cmdline-turbo-1.2.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/animetosho/par2cmdline-turbo"
+else
+	SRC_URI="https://github.com/animetosho/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="par2cmdline × ParPar: speed focused par2cmdline fork"
+HOMEPAGE="https://github.com/animetosho/par2cmdline-turbo"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="openmp"
```

Comment:
I know *toolchain-funcs.eclass* still has some function related to this, but I wonder if this is unnecessary in the ebuild? Let me know and I'll remove it on-merge if you agree.

Actually, I'll just remove it. It doesn't seem to do anything even if enabled. Add it back if you need it :)